gym_xiangqi/sound.py

```diff
@@ -0,0 +1,32 @@
+import pygame
+from gym_xiangqi.constants import PATH_TO_SOUNDS
+
+
+class Sound:
+    """
+    A class for sounds
+    """
+    def __init__(self):
+
+        # declare variables
+        self.bgm = None
+        self.piece_move = None
+
+        # init pygame mixer
+        pygame.init()
+        pygame.mixer.init()
+
+        # load sound effet for piece movements
+        self.load_piece_move()
+
+    # load background music
+    def play_bgm(self):
+        filename = "bgm.wav"
+        pygame.mixer.music.load(PATH_TO_SOUNDS + filename)
+        pygame.mixer.music.play(-1)
+
+    # load the sound effect for piece movements
+    def load_piece_move(self):
+        filename = "piece_move.wav"
+        target_file = PATH_TO_SOUNDS + filename
+        self.piece_move = pygame.mixer.Sound(target_file)
```

Comment:
- line 10: blank line can be omitted
- line 24: filename can be saved as class attribute
- line 30: filename can be saved as class attribute

To be honest, line 24 and 30 do not really matter, but the method `play_bgm` does not really depend on the `Sound` class anymore (i.e. it could be just a standalone function outside the class). But I like the idea of keeping code related to sound effects together so it would be nicer to add at least a small dependency between the `Sound` class and the methods we have.

Reply:
fixed. I'm a bit not sure if I correctly fixed them based your points, but I tried to simplified it. Please check it out!